superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/utilities.js

```diff
@@ -353,7 +370,7 @@ const baseAggregatorTemplates = {
                 throw new Error('unknown mode for runningStat');
             }
           },
-          format: formatter,
+          format: x => (typeof x === 'string' ? x : formatter(x)),
```

Comment:
Nit: since we do the same operation in line 241, could you extract that logic to a function?